Apps/Sandcastle/gallery/Per-Feature Post Processing.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Post processing effects.">
+    <meta name="cesium-sandcastle-labels" content="Showcases, Post Processing">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    shouldAnimate : true
+});
+
+var position = Cesium.Cartesian3.fromDegrees(-123.0744619, 44.0503706);
+var url = '../../SampleData/models/CesiumMan/Cesium_Man.glb';
+viewer.trackedEntity = viewer.entities.add({
+    name : url,
+    position : position,
+    model : {
+        uri : url
+    }
+});
+
+var collection = viewer.scene.postProcessStages;
+var silhouette = collection.add(Cesium.PostProcessStageLibrary.createSilhouetteStage());
+silhouette.uniforms.color = Cesium.Color.LIME;
+var blackAndWhite = collection.add(Cesium.PostProcessStageLibrary.createBlackAndWhiteStage());
+blackAndWhite.uniforms.gradations = 5.0;
+
+if (!silhouette.isSupported(viewer.scene)) {
```

Comment:
Isn't it odd that a user needs to create an instance to check `isSupported`?  Is there clean and obvious way to avoid this with static `isSupported` somewhere appropriate like `GroundPrimitive.isSupported`?

Reply:
How else would it work? There isn't a class for each of the stages. If there was, we could do something like `Silhouette.isSupported`.

Reply:
Maybe:
```javascript
if (!Cesium.PostProcessStageLibrary.isSilhouetteSupported(viewer.scene)) {
```

Slightly more coupled, but cleaner for the end user.

Reply:
@bagnell do you plan on adding this in https://github.com/AnalyticalGraphicsInc/cesium/pull/6680?

Reply:
Yup, db5cb75